src/RemoteInboxNotifications/StoredStateSetupForProducts.php

```diff
@@ -18,8 +18,6 @@ class StoredStateSetupForProducts {
 	 * Initialize the class
 	 */
 	public static function init() {
-		add_action( 'product_page_product_importer', array( __CLASS__, 'run_on_product_importer' ) );
```

Comment:
Again wondering if we should modify the methods being called here, allow them to do some of the logic, but stop short of running the call to `RemoteInboxNotificationsEngine::run`